domain/src/main/kotlin/app/discord/user/dto/UserRegisterEvent.kt

```diff
@@ -2,9 +2,11 @@ package app.discord.user.dto
 
 import java.time.OffsetDateTime
 
-data class UserRegisterEvent(
+class UserRegisterEvent(
```

Comment:
별다른 이유가 없다면 data클래스로 다시 변환해주세요.
```suggestion
data class UserRegisterEvent(
```